base/system/diskpart/lang/pl-PL.rc

```diff
@@ -214,11 +214,11 @@ BEGIN
     IDS_HELP_RETAIN                    "Umieść zachowaną partycję w woluminie prostym.\n"
     IDS_HELP_SAN                       "Wyświetla lub ustawia zasady SAN dla aktualnie uruchomionego\n              systemu operacyjnego.\n"
 
-    IDS_HELP_SELECT                    "Przenieś fokus na jeden z obiektów.\n"
-    IDS_HELP_SELECT_DISK               "Moves the focus to the disk.\n"
-    IDS_HELP_SELECT_PARTITION          "Moves the focus to the partition.\n"
-    IDS_HELP_SELECT_VOLUME             "Moves the focus to the volume.\n"
-    IDS_HELP_SELECT_VDISK              "Moves the focus to the virtual disk.\n"
+    IDS_HELP_SELECT                    "Przenieś fokus na obiekt.\n"
+    IDS_HELP_SELECT_DISK               "Przenieś fokus na dysk.\n"
+    IDS_HELP_SELECT_PARTITION          "Przenieś fokus na partycję.\n"
+    IDS_HELP_SELECT_VOLUME             "Przenieś fokus na wolumin.\n"
+    IDS_HELP_SELECT_VDISK              "Przenieś fokus na dysk wirtualny.\n"
```

Comment:
I'm not sure about it. Can you send screenshot from PL WinXP?

Reply:
Seems fine to me. Search "Move focus" on MS Language Portal.